projects/pokemon_game.py

```diff
@@ -75,65 +71,87 @@ def feed(self):
     def __str__(self) -> str:
         return f"{self.name} {self.primary_type} {self.max_hp} {self.hp}"
 
+charmander = Pokemon(
```

Comment:
**suggestion**: It's generally best to keep this type of "configuration" data outside of your script. This might be a good chance to brush up on your csv and file manipulation.

I'd like you to have a csv file:

```csv
Charmander,fire,50,50
Snivy,grass,60,60
Sobble,water,60,60
```

Which your app will read and dynamically create the Pokemon. I suggest ending up with a dictionary of pokemon.